### PR TITLE
Intro section

### PR DESCRIPTION
**Pull request by LalightKosh**
This is a branch with "intro" section changes from [this PR](https://github.com/totfront/somewhere-travel/pull/5)

### CODE REVIEW

File: src/components/Lead/Lead.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ function Lead() {
         Настоящая страна не в выпусках новостей, а здесь.
       </subtitle>
       <figure className={styles.figure}>
-        <figcaption className={styles.caption}>ваша полка — верхняя</figcaption>
+        <figcaption className={styles.caption}>ваша полка - верхняя</figcaption>
```
Comment (totfront) [inline src/components/Lead/Lead.js:12]: According to the prototype there should be "—"

**Review by totfront — CHANGES_REQUESTED** [review]
The review is rejected:

1. There are conflicts with the main branch that should be resolved
2. Only mobile prototype has been implemented, but not the rest

Please, fix those two and request again

**Review by totfront — APPROVED** [review]

**Comment by totfront** [comment]
Please always check if the build is fine. This is critically important.
image

**Comment by totfront** [comment]
Nice and clean PR. That was a pleasure to check it. Good work with media queries and good-looking small CSS files.

Design inconcistancies:
1. Space before "часовых поясов" should be 34px. You have not explicitly set it and not it is occasionally made of 2 margins 16px + 16px.

I would like you to fix all of my suggestions, but they are slightly out of the requirements except the one with the design. Fix it and I will aprove.

**Comment by LalightKosh** [comment]
> Please always check if the build is fine. This is critically important. image

It is working for me

**Comment by LalightKosh** [comment]
> Nice and clean PR. That was a pleasure to check it. Good work with media queries and good-looking small CSS files.
> 
> Design inconcistancies:
> 
> 1. Space before "часовых поясов" should be 34px. You have not explicitly set it and not it is occasionally made of 2 margins 16px + 16px.
> 
> I would like you to fix all of my suggestions, but they are slightly out of the requirements except the one with the design. Fix it and I will aprove.

done.

**Comment by totfront** [comment]
The issue is this there. Please, fix it

**Comment by LalightKosh** [comment]
> 

I fixed margins.

**Review by totfront — APPROVED** [review]

File: src/components/Intro/Intro.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import styles from "./Intro.module.css";
+
+function Intro() {
+  return (
+    <section className={styles.wrapper}>
+      <title className={styles.title}>Чего мы там не видели?</title>
```
[the remark this answers is not in the document]
Comment (LalightKosh) [reply src/components/Intro/Intro.js:6]: changed. I have changed it everywhere already, but forgot about Intro.

File: src/components/Intro/Intro.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      </p>
+      <ul className={styles.introList}>
+        <li className={styles.introItem}>
+          <span className={styles.boldItem}>Часовых поясов</span> 11
+        </li>
+        <li className={styles.introItem}>
+          <span className={styles.boldItem}>
+            Объектов природного наследия ЮНЕСКО
+          </span>{" "}
+          12
```
[the remark this answers is not in the document]
Comment (LalightKosh) [reply src/components/Intro/Intro.js:26]: I have changed this on a different branch and merged. :/ sorry, got my lesson.

File: src/components/Intro/Intro.module.css [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        max-width: 690px;
+        align-items: start;
+        margin: 64px 286px 0 48px
+    }
+}
+
+@media (min-width: 1280px) {
+    .wrapper {
+        max-width: 690px;
+        align-items: start;
```
[the remark this answers is not in the document]
Comment (LalightKosh) [reply src/components/Intro/Intro.module.css:65]: you are correct. I have removed flex.

File: src/components/Intro/Intro.module.css [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    font-size: 16px;
+    line-height: 26.08px;
+}
+
+.introList {
+    display: inline-block;
+    padding-inline-start: 0px;
+    list-style: none;
+    align-self: flex-start;
+    margin-left: 34px;
```
[the remark this answers is not in the document]
Comment (LalightKosh) [reply src/components/Intro/Intro.module.css]: fixed

**Pull request merged**